engine/consensus/dkg/messaging_engine.go

```diff
@@ -77,27 +77,32 @@ func (e *MessagingEngine) Done() <-chan struct{} {
 // SubmitLocal implements the network Engine interface
 func (e *MessagingEngine) SubmitLocal(event interface{}) {
 	e.unit.Launch(func() {
-		err := e.Process(engine.DKGCommittee, e.me.NodeID(), event)
-		if err != nil {
-			engine.LogError(e.log, err)
+		err := e.process(e.me.NodeID(), event)
```

Comment:
Sorry, my last suggestion did not fully cover these cases. 

We are getting an error from processing either a local or remote message. In either case, we want to crash in the case of an unexpected error, and handle expected errors.

For remote messages, the only expected error is `IsInvalidInputError`. We are handling it, but we should also crash for any other, unexpected errors (line 90-92).
Eg add:
```
else if err != nil {
  // crash
}
```

For local messages, there is no expected error. Rather than crashing only in the case of one explicitly disallowed error type, we can crash for any error (lines 81-83)
Eg change to:
```
if err != nil {
  // crash
```